modules/Bio/EnsEMBL/Variation/TranscriptVariationAllele.pm

```diff
@@ -1417,15 +1417,15 @@ sub hgvs_transcript {
   my @edit_attrs = grep {$_->code =~ /^_rna_edit/} @attribs;
 
   my $misalignment_offset = 0;
-  $misalignment_offset = $self->get_misalignment_offset(\@edit_attrs) if (scalar(@edit_attrs) && substr($tr->stable_id, 0,3) eq 'NM_');
+  $misalignment_offset = $self->get_misalignment_offset(\@edit_attrs) if (scalar(@edit_attrs) && (substr($tr->stable_id, 0,3) eq 'NM_' || substr($tr->stable_id, 0,3) eq 'XM_'));
   
   if ($vf->var_class eq 'SNP' && defined($self->{pre_consequence_predicates}) && $self->{pre_consequence_predicates}->{exon} && defined($tv->cds_start) && defined($tv->cds_end)) {
```

Comment:
Currently the test sets `{pre_conseuqence_predicates}->{exon}` which triggers this if. Where is this value expected to be set if a pipeline is run?

Reply:
Thanks, I've added a comment